src/implementations/deploy/token-registry/token-registry.ts

```diff
@@ -56,13 +56,21 @@ export const deployTokenRegistry = async ({
   trace("[Status] Title Escrow Factory interface check is OK.");
 
   info(`Using ${factoryAddress} as Title Escrow factory.`);
+
+  // Blockchains where deployers are not supported.
+  if (!isValidAddress(deployerContractAddress)) {
+    deployerContractAddress = defaultDeployerContractAddress;
+  }
+  if (!isValidAddress(implAddress)) {
+    implAddress = defaultTokenImplementationContractAddress;
+  }
+
+  if ((!deployerContractAddress || !implAddress) && !standalone) {
+    console.error(`Network ${chainId} does not support "quick-start" mode. Defaulting to --standalone mode.`);
+    standalone = true;
+  }
```

Comment:
Can we show the error message but don't proceed with the default standalone behaviour straightaway? That's to say just display the error suggesting the user use the `--standalone` mode because quick start mode is not available.

Reply:
Thanks for the update. Can I check what does the additional `configDeploy` option mean if supplied? Is there a configuration somewhere that the user has to do?